src/features/click-to-load.js

```diff
@@ -617,7 +617,11 @@ function showExtraUnblockIfShortPlaceholder (shadowRoot, placeholder) {
     if (!placeholder.parentElement) {
         return
     }
-
+    const parentStyles = window.getComputedStyle(placeholder.parentElement)
+    // Inline elements, like span or p, don't have a set height that we can trust.
+    if (parentStyles.display === 'inline') {
```

Comment:
What happens when there's nested spans?
Do we need to traverse to the nearest non inline element for height instead?

Reply:
Yea, this is a fair point :+1: .

I have tested quite a bit today though and Fran's changes make the placeholders display far more reliably. Suggest we go with this for now, and expand to cover edge-cases like that if it proves to be an issue in practice.

Reply:
(Fran, in the mean time perhaps add a comment to note Jonathan's point? It might be useful to expand the logic to consider grandparent etc elements in the future.)

Reply:
I was also thinking about that too, @jonathanKingston, although it might have other implications. As we go up the DOM tree, there are more elements being involved and the styles might not be as straightforward to verify. I think we can start with this and observe other reports for examples where this is not enough.

Either way, I'll add a comment so we know what to address if necessary.

Reply:
done